src/workerd/api/streams/internal.c++

```diff
@@ -1426,6 +1462,17 @@ jsg::Promise<void> WritableStreamInternalController::writeLoopAfterFrontOutputLo
         finishError(js, handle);
       }));
     }
+    KJ_CASE_ONEOF(request, Flush) {
+      // Flushing is similar to closing the stream, the main difference is that `finishClose`
+      // and `writable->end()` are never called.
```

Comment:
I would indicate that this is not a standards-defined state for a WritableStream in case someone later goes trying to look it up in the spec.